app/panel/components/Settings/TrustAndRestrict.jsx

```diff
@@ -86,9 +86,10 @@ class TrustAndRestrict extends React.Component {
 	/**
 	 * Implement adding site to the list of whitelisted or blacklisted sites
 	 * This routine valides entered url and checks if the entered url is a duplicate, or
-	 * if it has been alreday added to the opposite list. Displays appropriate warnings.
+	 * if it has been already added to the opposite list. Displays appropriate warnings.
 	 */
 	addSite() {
+		// TODO: Implement paygating, use this regex for validating free users who don't have regex/wildcard ability
```

Comment:
Just curious as to why we aren't implementing the paygating in this PR?

Reply:
Chris said don't worry about it for now. I think he wanted me to focus on implementation first.

Reply:
Ah okay, then let's update the ticket to reflect that — unless @christophertino you would like me to pair up with @benstrumeyer and knock out the paygating real quick? It should be pretty straightforward since we have the account/subscription info in the `AccountReducer`.

Reply:
@benstrumeyer Looks like @christophertino doesn't want this to be a Plus feature anyway, and he said he'd update the ticket to refelect that.